### DISCUSSION TITLE
Questions about Sample Data 202107

### DISCUSSION THREAD

**Discussion by wzgytuA888** [WAM2layers/WAM2layers#255] [Q&A] [answered]
![image](https://github.com/WAM2layers/WAM2layers/assets/128116960/b755afcd-aab0-4b2b-97d8-c0d6874b0ae4)
By running the sample data, it was found that there were results, but the results were all null values and not of the type of NC file. May I know the reason for this

**Comment by Peter9192** [comment]
Could it be that you're computer is having some problems with the semicolons in the filenames? Can you try to open one of those files with a xarray, ncdump, panoply or any other tool that can read netcdf files?

**Reply by wzgytuA888** [reply]
Are you referring to a sample dataset? I used wam2layers to run the program according to the user guide, and the result was an empty file with only the name, and the file type was not in NC format. The memory was 0kb
The same result was achieved when using Conda. How can I solve this problem
![image](https://github.com/WAM2layers/WAM2layers/assets/128116960/69d10a31-1aa7-4c6d-9f79-bde66dd74bbb)

**Reply by Peter9192** [reply]
Do you know how to open a netcdf file? Please try out some of the tools I suggested.

**Reply by Peter9192** [reply]
Don't trust the windows file type. Also, you can't open a netcdf file by simply (double) clicking it. I suggest you first try to open one of the sample files to verify that you can indeed open them with one of the tools I suggested. Then try the same with the output. I bet the files are just fine but your windows is confused due to the colon in the file names. (see the log you posted: the output filenames are like `backtrack_2021-07-01T00:00.nc` but windows explorer cut's them off at the colon

**Reply by wzgytuA888** [reply]
Thank you very much for your answer. I will try my best to give your suggestion a try

**Reply by Peter9192** [reply, marked as answer]
I created a fix for the filename here: https://github.com/WAM2layers/WAM2layers/pull/257
It will take some time for this to be available on PyPI though. If you want to try it out in the meantime, you could try installing with 
`pip install git+https://github.com/WAM2layers/WAM2layers.git@no-colons#egg=wam2layers`
